src/services/clinical/index.js

```diff
@@ -162,6 +162,21 @@ const getClinicalData = async (variables, Authorization) => {
   return response;
 };
 
+const getClinicalSearchResults = async (variables, Authorization) => {
+  const { programShortName, filters } = variables;
+
+  const query = new URLSearchParams(filters).toString();
+
+  const url = `${CLINICAL_SERVICE_ROOT}/clinical/program/${programShortName}/clinical-search-results?${query}`;
```

Comment:
would UrlJoin work here?

Reply:
It does work would you like it added here + on Clinical Search Results / Clinical Error endpoints?

Reply:
yes please, we run into problems with slashes in URLs sometimes 👍

Reply:
Updated